build/scripts/GuardingV2ExtensionsHelper.psm1

```diff
@@ -43,6 +43,22 @@ function Enable-BreakingChangesCheck {
             }
             $baselineVersion = $appJson.version # Use the version of the current app as the baseline version
         }
+        'Strict' {
+            Write-Host "Getting baseline for Strict Mode"
+            Import-Module -Name $PSScriptRoot\EnlistmentHelperFunctions.psm1
+
+            $majorMinor = Get-ConfigValue -Key "repoVersion" -ConfigType "AL-GO"
+            $strictModeVersion = ((Get-BCArtifactUrl -type Sandbox -country W1 -version $majorMinor -select Latest) -split "/")[-2]
+            if (-not $strictModeVersion) {
+                $strictModeVersion = ((Get-BCArtifactUrl -type Sandbox -country W1 -version $majorMinor -select Latest -storageAccount bcinsider -accept_insiderEula) -split "/")[-2]
```

Comment:
Strict mode should only run on release branches, meaning that the baselines should also be released, so they should already be in _bcartifacts_ and not _bcinsider_.